lib/chef/formatters/doc.rb

```diff
@@ -80,18 +80,18 @@ def run_completed(node)
           puts_line ""
         end
         if Chef::Config[:why_run]
-          puts_line "#{ChefUtils::Dist::Infra::PRODUCT} finished, #{@updated_resources}/#{total_resources} resources would have been updated"
+          puts_line "Infra Phase complete, #{@updated_resources}/#{total_resources} resources would have been updated"
```

Comment:
`Infra Phase` seems a bit weird to me, `Configuration Phase` would have been what i would go for.  That way you have Configuration/Compliance as opposed to branding with Infra/InSpec.

Reply:
(you're product guy tho, up to you, just seems odd in my brain)

Reply:
The split isn't as much what it's doing but the traditional concern it's in control of. Infra vs. Compliance. It's why we don't have a scanning phase. It's mostly a marketing change to signal the capabilities of the product being infra + compliance now not just traditional config management.